comps/comps-foreman-fedora21.xml

```diff
@@ -144,17 +144,17 @@
       <packagereq type="default">rubygem-hirb-doc</packagereq>
       <packagereq type="default">rubygem-hirb-unicode-doc</packagereq>
       <packagereq type="default">rubygem-i18n-doc</packagereq>
-      <packagereq type="default">rubygem-jquery_pwstrength_bootstrap_4-doc</packagereq>
       <packagereq type="default">rubygem-jquery-turbolinks-doc</packagereq>
       <packagereq type="default">rubygem-jquery-ui-rails-doc</packagereq>
+      <packagereq type="default">rubygem-jquery_pwstrength_bootstrap_4-doc</packagereq>
```

Comment:
Ah, that appears to be an en_* thing.  en_GB and en_US sort hyphens before underscores, while C sorts it the other way around.